src/clj/game/core/runs.clj

```diff
@@ -697,7 +697,7 @@
     (:successful run)
     (do
       (play-sfx state side "run-successful")
-      (trigger-event-sync state side eid :successful-run-ends run))
+      (trigger-event-simult state side eid :successful-run-ends nil run))
```

Comment:
Why this change?

Reply:
It was a part solution to handling the interaction between Doppelganger + Data Breach/Mobius.  I have been thinking about that today and think those cards just need a re-write such they use the same EID for their prompt and the run they actually fire.

Reply:
Gotcha 👍

Reply:
Figured it out - booyah!